plugins/datalogger/src/datalogger.cpp

```diff
@@ -1,60 +1,57 @@
 #include "datalogger.hpp"
-#include "gui/generic_menu.hpp"
-#include "gui/customSwitch.hpp"
-#include "gui/detachedwindowsmanager.h"
-#include "gui/dynamicWidget.hpp"
+#include "dataloggerplugin.hpp"
+#include "src/dataloggercontroller.hpp"
 #include <QtConcurrent>
 #include <QFuture>
+#include <tool_view_builder.hpp>
+#include <src/gui/channelmonitorcomponent.hpp>
```

Comment:
instead of #include <src/gui/channelmonitorcomponent.hpp> you can use #include "gui/channelmonitorcomponent.hpp", and so on ..